tests/Mocks/MockDirectEventCapture.php

```diff
@@ -26,6 +26,6 @@ protected function fetch(string $method, string $url, string $body, array $heade
     {
         $this->requests[] = new MockRequest($body, $headers);
 
-        return "";
+        return json_encode(['id' => 1]);
```

Comment:
Was a test breaking before this change?

Reply:
Nothing breaking no, it was just catching all the errors here https://github.com/kodus/sentry/blob/master/src/Model/DirectEventCapture.php#L55
and printing them in the console creating a lot of bloated output.

So I thought we might as well simulate what Sentry does with the backed IDs since it's just a mock.